packages/vue/src/components/organisms/SfHero/SfHero.vue

```diff
@@ -0,0 +1,5 @@
+<script src="./SfHero.js"></script>
+<template lang="html" src="./SfHero.html"></template>
+<style lang="scss">
+@import "../../../../../shared/styles/components/SfHero.scss"
```

Comment:
The reason I've chosen to use `SfCarousel` is to follow the principles of atom design, so that organisms are built from molecules and atoms. Correct me if I'm wrong, but here `SfCarousel` is similar to designed `SfHero` - basically it is a carousel with only one visible picture and the pagination. 

I am not gonna argue that it is the best path, because I had a hard time to decide how to approach it myself.

Basically I wonder if:
1. maybe  `SfCarousel` should be more flexible to used in `SfHero`, or as you suggested create a new organism/molecule `SfHero` "from scratch".
2. `SfArrow` should be extended with styling that allows the `transparent` background.